src/calculateRentalCost.test.js

```diff
@@ -3,21 +3,17 @@
 const calculateRentalCost = require('./calculateRentalCost');
 
 test(`Do not add any discount for less than 3 days`, () => {
-  expect(calculateRentalCost(2))
-    .toBe(80);
+  expect(calculateRentalCost(2)).toBe(80);
 });
 
 test('Should add the basic discount from 3 to 6 days of rent', () => {
-  expect(calculateRentalCost(3))
-    .toBe(100);
+  expect(calculateRentalCost(3)).toBe(100);
 });
 
 test('Should add the basic discount from 3 to 6 days of rent', () => {
-  expect(calculateRentalCost(6))
-    .toBe(220);
+  expect(calculateRentalCost(6)).toBe(220);
 });
```

Comment:
The description for the test case on line 13 should be unique. It is currently the same as the test case on line 9. Consider changing it to reflect that this test is specifically for the upper bound of the basic discount range.